### PR TITLE
[Refactor]Refactor config structure

### PR DESCRIPTION
**Pull request by congee524**
Refactor config structure

I passed all the unittests and wrote a check function to determine the equivalence between the new config file and the old one

### CODE REVIEW

**Comment by dreamerlin** [comment]
@xvjiarui can you help to review it ?

File: configs/_base_/models/tsn_r18.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# model settings
+model = dict(
+    type='Recognizer2D',
+    backbone=dict(
+        type='ResNet',
+        pretrained='torchvision://resnet18',
+        depth=18,
```
Comment (xvjiarui) [inline configs/_base_/models/tsn_r18.py:7]: `r18` model may not be included in `_base_` since there is `r50` one.

Comment (congee524) [reply configs/_base_/models/tsn_r18.py:7]: okk

**Comment by congee524** [comment]
kindly ping :p

**Review by xvjiarui — APPROVED** [review]

**Review by hellock — APPROVED** [review]

**Pull request merged**